task_4/StoynovOleg/src/main/java/Four.java

```diff
@@ -0,0 +1,29 @@
+package maine.java;
+
+public class Four {
+  public static void main(String... args) {
+    int sum = Integer.parseInt(args[0]);
+    String operation = args[1];
+    int result = 0;
+    switch (operation) {
+      case "terrible":
+          result = (sum / 100) * 0;
```

Comment:
You don't have to calculate the result which is going to be 0 in any case
You can write:
result = 0;